contracts/burners/UniswapLPBurner.vy

```diff
@@ -0,0 +1,197 @@
+# @version 0.2.12
+"""
+@title Uniswap LP Burner
+@notice Given a Uniswap LP token, withdraw from the pool, and send the withdrawed two tokens to next level burner
+"""
+
+from vyper.interfaces import ERC20
+
+
+interface UniswapV2Pair:
+    def token0() -> address: view
+    def token1() -> address: view
+    def factory() -> address: view
+
+interface UniswapV2Router02:
+    def removeLiquidity(tokenA:address, tokenB:address, liquidity:uint256, amountAMin:uint256, amountBMin:uint256, to:address, deadline:uint256) -> uint256[2]: nonpayable
+    def factory() -> address: view
+
+recovery: public(address)
+is_killed: public(bool)
+
+owner: public(address)
+emergency_owner: public(address)
+future_owner: public(address)
+future_emergency_owner: public(address)
+next_level_burner: public(address)
+routers: constant(address[2]) = [
+    0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D, # uniswap
+    0xd9e1cE17f2641f24aE83637ab66a2cca9C378B9F # sushiswap
+]
+
+
+
+event Burn:
+    lp_token: address
+    amount: uint256
+    token0_amount:uint256
+    token1_amount:uint256
+
+@external
+def __init__(_recovery: address, _owner: address, _emergency_owner: address, _next_level_burner:address):
```

Comment:
stay consistent with other burners
 add `_receiver` and remove `_next_level_burner`